sqlite-simple.cabal

```diff
@@ -23,7 +23,7 @@ Category:            Database
 Build-type:          Simple
 
 Cabal-version:       >= 1.10
-
+tested-with:         GHC == 8.8.4 || == 8.10.7 || == 9.0.2 || == 9.2.8 || == 9.4.8 || == 9.6.3 || == 9.8.1
```

Comment:
You can check in my repo that CI is passing for all these ghc versions (https://github.com/jhrcek/sqlite-simple/actions/runs/7472016831/job/20333355139?pr=10). The CI would start working in this repo after this PR is merged.